tests/ClassReferencesProcessTest.php

```
@@ -85,6 +85,16 @@ public function can_detect_class_references()
         $this->assertCount(0, $attributeRefs);
     }
 
+    /** @test */
+    public function can_detect_anonymous_class_references()
+    {
+        $tokens = $this->getTokens(__DIR__.'/stubs/anonymous_class_references.stub');
+        [$classRefs, $namespace, $attributeRefs] = ClassReferenceFinder::process($tokens);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$namespace'. (undefined)

(UnusedLocalVariable)